### DISCUSSION TITLE
Reset Password Using OTP Wihtout any Links

### DISCUSSION THREAD

**Discussion by AbrarAlHasan** [supabase/supabase#29353] [Feature Requests]
When Implementing the Restting of Password Logic, In React Native it is very hard and so complex to handle it. So a Service should allow to send OTP to the Registered Email Address and verify the OTP in the Client Side.

**Comment by pmilaszewski-asper** [comment]
Totally agree, for mobile apps without website it's almost impossible to reset / change password in any way. Using OTP would be a great solution to verify so user can provide 6-digit code + password to function on client side and change password.
